intera_examples/scripts/set_interaction_options.py

```diff
@@ -0,0 +1,201 @@
+                                     description=main.__doc__)
+    parser.add_argument(
+        "-s",  "--interaction_active", type=int, default=1, choices = [0, 1],
+        help="Activate (1) or Deactivate (0) interaction controller")
+    parser.add_argument(
+        "-k", "--K_impedance", type=float,
+        nargs='+', default=[1300.0, 1300.0, 1300.0, 30.0, 30.0, 30.0],
+        help="A list of desired stiffnesses, one for each of the 6 directions -- stiffness units are (N/m) for first 3 and (Nm/rad) for second 3 values")
+    parser.add_argument(
+        "-m", "--max_impedance", type=int,
+        nargs='+', default=[1, 1, 1, 1, 1, 1], choices = [0, 1],
+        help="A list of maximum stiffness behavior state, one for each of the 6 directions (a single value can be provided to apply the same value to all the directions) -- 0 for False, 1 for True")
+    parser.add_argument(
+        "-md", "--interaction_control_mode", type=int,
+        nargs='+', default=[1, 1, 1, 1, 1, 1], choices = [1,2,3,4],
+        help="A list of desired interaction control mode (1: impedance, 2: force, 3: impedance with force limit, 4: force with motion limit), one for each of the 6 directions")
+    parser.add_argument(
+        "-fr", "--interaction_frame", type=float,
+        nargs='+', default=[0, 0, 0, 1, 0, 0, 0],
+        help="Specify the reference frame for the interaction controller -- first 3 values are positions [m] and last 4 values are orientation in quaternion (w, x, y, z) which has to be normalized values")
+    parser.add_argument(
+        "-ef",  "--in_endpoint_frame", action='store_true', default=False,
+        help="Set the desired reference frame to endpoint frame; otherwise, it is base frame by default")
+    parser.add_argument(
+        "-en",  "--endpoint_name", type=str, default='right_hand',
+        help="Set the desired endpoint frame by its name; otherwise, it is right_hand frame by default")
+    parser.add_argument(
+        "-f", "--force_command", type=float,
+        nargs='+', default=[0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
+        help="A list of desired force commands, one for each of the 6 directions -- in force control mode this is the vector of desired forces/torques to be regulated in (N) and (Nm), in impedance with force limit mode this vector specifies the magnitude of forces/torques (N and Nm) that the command will not exceed")
+    parser.add_argument(
+        "-kn", "--K_nullspace", type=float,
+        nargs='+', default=[4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0],
+        help="A list of desired nullspace stiffnesses, one for each of the 7 joints (a single value can be provided to apply the same value to all the directions) -- units are in (Nm/rad)")
+    parser.add_argument(
+        "-r",  "--rate", type=int, default=10,
+        help="A desired publish rate for updating interaction control commands (10Hz by default) -- 0 if we want to publish it only once")
+
+    args = parser.parse_args()
```

Comment:
- [ ] you should use the special ros-argument-aware format of parsing the args. see old examples or scripts. (last time I checked, that was still needed to not break on ros := or param definitions)